src/dsp/sniffer_f.cpp

```diff
@@ -101,7 +101,7 @@ int  sniffer_f::samples_available()
 /*! \brief Fetch available samples.
  *  \param out Pointer to allocated memory where the samples will be copied.
  *             Should be at least as big as buffer_size().
- *  \param num The number of sampels returned.
+ *  \param num The number of samplss returned.
```

Comment:
```suggestion
 *  \param num The number of samples returned.
```

Reply:
Weird, the github UI has greyed out the button that merges this change. :confounded:

Reply:
That's because I merged it first. :smile: